.scripts/enumerate-product-versions.py

```diff
@@ -1,3 +1,9 @@
+import sys
+import os
+
+# Add parent directory to the sys.path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Does it need to be the parent dir or should it be the CWD?

We did the CWD version here: https://github.com/stackabletech/image-tools/pull/27/files

```suggestion
# Add CWD directory to the sys.path
sys.path.append(str(os.getcwd()))
```

Reply:
"my" version works regardless of whether you call the script from / or from ./scripts.
the cwd version only works if being called from /

but the cwd version will continue working as long as the script is always called from the folder where `conf.py` resides, my version will continue working as long as the relative location of the two files stays the same (parent/child)

I'd say both are not perfect and there's no clear winner.

Reply:
If it works :shrug:

Reply:
@lfrancke, I checked locally, and indeed the cwd way works.

![image](https://github.com/user-attachments/assets/e3b6efde-eaa1-4f34-a578-c2ee51630571)

```diff
diff --git a/.scripts/enumerate-product-versions.py b/.scripts/enumerate-product-versions.py
index d4afe83..08c29e7 100644
--- a/.scripts/enumerate-product-versions.py
+++ b/.scripts/enumerate-product-versions.py
@@ -1,3 +1,8 @@
+import os
+import sys
+
+sys.path.append(str(os.getcwd()))
+
 import conf
 
 PRODUCTS = [
```

Reply:
Yes. I know. But it only works if called from `/` (the root directory of the repo)

The other solutions also works when you e.g. call it from within the `.scripts` directory while the `cwd` version doesn't.